ted2zim/entrypoint.py

```diff
@@ -141,6 +141,20 @@ def main():
         dest="source_language",
     )
 
+    parser.add_argument(
+        "--subtitles-enough",
+        help="Consider subtitle availability while filtering videos by language",
+        default=False,
+        action="store_true",
+    )
+
+    parser.add_argument(
```

Comment:
Actually no. Because in the current version, it also takes in exact language codes to have subtitles in only those languages. Say, for example `--subtitle="en, ja, fr, es"`

Reply:
oh right. ok